### ISSUE TITLE
Colocate `config` as a peer to `src` (not under `src`).

### ISSUE THREAD

**Issue by rwjblue** [glimmerjs/glimmer-application-pipeline#72]
In the current `@glimmer/blueprint` we are assuming that config is nested underneath `src` ([see here](https://github.com/glimmerjs/glimmer-blueprint/blob/04a3e7ef5c17d21fa4772a606c6fed96bb39867f/files/src/main.ts#L3-L4), but that is not true and ultimately makes things fairly confusing (IMHO).

We should update the various bits of the pipeline to ensure that the mental model is:

```
├── config
│   ├── environment.js
│   ├── module-map.d.ts
│   ├── module-map.js
│   └── resolver-configuration.js
└── src
    ├── index.js
    └── ui
        ├── components
        │   └── foo-bar.js
        └── index.html
```

**Comment by rwjblue** [comment]
Note that this is a somewhat breaking change for the default `@glimmer/blueprint`, the following change is needed for users to upgrade:

```diff
diff --git a/src/config/module-map.ts b/src/config/module-map.ts
deleted file mode 100644
index 0794fcb..0000000
--- a/src/config/module-map.ts
+++ /dev/null
@@ -1,6 +0,0 @@
-/**
- * This is just a placeholder file to keep the TypeScript compiler happy. At build time, 
- * it will be replaced with a complete map of resolvable module paths => rolled up contents.
- */
-
-export default {};
diff --git a/src/config/resolver-configuration.ts b/src/config/resolver-configuration.ts
deleted file mode 100644
index 4e966af..0000000
--- a/src/config/resolver-configuration.ts
+++ /dev/null
@@ -1,9 +0,0 @@
-/**
- * This is just a placeholder file to keep the TypeScript compiler happy. At build time, 
- * it will be replaced with a resolver configuration composed from your application's
- * `config/environment.js` (and supplemented with default settings as possible).
- */
-
-import { ResolverConfiguration } from '@glimmer/resolver';
-
-export default <ResolverConfiguration>{};
diff --git a/src/main.ts b/src/main.ts
index eb53f17..cb0366f 100644
--- a/src/main.ts
+++ b/src/main.ts
@@ -1,7 +1,7 @@
 import Application from '@glimmer/application';
 import Resolver, { ResolverConfiguration, BasicModuleRegistry } from '@glimmer/resolver';
-import moduleMap from './config/module-map';
-import resolverConfiguration from './config/resolver-configuration';
+import moduleMap from '../config/module-map';
+import resolverConfiguration from '../config/resolver-configuration';
 
 export default class App extends Application {
   constructor() {
@@ -13,4 +13,4 @@ export default class App extends Application {
       resolver
     });
   }
-}
\ No newline at end of file
+}
```

**Comment by rwjblue** [comment]
This was fixed in #73 and released in 0.6.0.
